app/models/contract.rb

```diff
@@ -62,10 +62,15 @@ def self.issued_by(issuer, *ids)
     where.has { plan_id.in( scope ) }
   end
 
+  # FIXME: this probably shouldn't return `all`, it's dangerous if it is accidentally
```

Comment:
I don't really want to fix it in this PR as it is already too long. And it was already this way before...

Reply:
Maybe use scope `provider_account: user.provider_account` although if we don't have these fields and I think we don't, then it may be complicated. So for now it is not worse than before at least.